_drafts/randy/2022-07-17-10-second-video.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "How Can 10 Second of Video Take All Day?"
+author: randy
+permalink: /@randy/10-second-video-takes-all-day
+image: "assets/img/10-second-video.png"
```

Comment:
Looks like you forgot to add the image to /assets/img/10-second-video.png to this pull request.

Reply:
lost along the way

Reply:
wtf did it go?